### ISSUE TITLE
Broken links: http://deeplearning4j.org/lstm

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#1992]
"The learning rate is the single most important hyperparameter. Tune this using deeplearning4j-ui; see [this graph] (http://cs231n.github.io/neural-networks-3/#baby)"

Link to "deeplearning4j-ui" is broken. Also this might be better if it pointed to http://deeplearning4j.org/visualization

Last link is not rendering properly.

**Comment by ejunprung** [comment]
@AlexDBlack Fixed. I've pointed deeplearning4j-ui to http://deeplearning4j.org/visualization instead. Also fixed the markdown typo in the last link.
